applications/qserv-kafka/values.yaml

```diff
@@ -126,9 +126,17 @@ frontend:
   # -- Affinity rules for the qserv-kafka frontend pod
   affinity: {}
 
-  # -- Whether to allow containers to run as root. Set to true to allow use of
-  # debug containers to diagnose issues such as memory leaks.
-  allowRootDebug: false
+  # -- Settings for debugging
```

Comment:
You only want documentation strings on the individual settings, not on the containers, or the helm-docs output and defaults will be weird.